### ISSUE TITLE
[Dev Asks} anyone got a codex item pick up issue?

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5749]
I just played a bit in the normal player (I usually play from the Unity editor) and had a bug where it was clumsy to drag an item from the codex. I had to pick up something from the item bar first otherwise it would not work. Did someone got that issue? I did not managed to reproduce when restarting the game after.
Cheers

**Comment by king32109** [comment]
I had it a few times forgot to report it. But usualy it happens in a just loaded in workd and q restart fixes it

**Comment by ProPeach** [comment]
It's pretty interesting how often there are bugs in the Steam branch but not in the editor 🤔
I don't think I've experienced this one though, never had any issues dragging stuff. Maybe it could be related to the multiple brick shape icons? I very rarely use those, I always grab the individual bricks

**Comment by tsunamayo** [comment]
@ProPeach Yeah it is pretty rare but this sometime happen. I cant say it is one of those yet, might just be a coincidence.

**Comment by CoasterBlock** [comment]
Yes I do actally have that Issue! Sometimes I cant even pick items up from the codex when I copy/pasted something or used a tool. It isnt 100% reproducable, but thats where I noticed it the most. Switching to another tool or canceling a copy will usually fix it
